kano_updater/paths.py

```diff
@@ -14,3 +14,5 @@
 SYSTEM_VERSION_FILE = '/etc/kanux_version'
 
 STATUS_FILE_PATH = '/var/cache/kano-updater/status.json'
+INTERRUPTED_SPLASH_PATH = '/usr/lib/python2.7/dist-packages/kano_updater/ui/media/images/'
```

Comment:
Base directory available as [kano_updater.ui.paths.IMAGE_PATH](https://github.com/KanoComputing/kano-updater/blob/master/kano_updater/ui/paths.py#L15).

Reply:
Also bump copyright.

Reply:
I wondered why it wasn't there already. Okay, I've eliminated use of this file, so no need to bump copyright.